src/network/http.py

```diff
@@ -55,7 +59,7 @@ def state_socks_handshake_done(self):
 
 if __name__ == "__main__":
     # initial fill
-
+    # pylint: disable=len-as-condition
```

Comment:
here also probably the conditions can be rewritten rather than using `len` and then this line removed.